modules/ROOT/pages/questions/question-13.adoc

```diff
@@ -0,0 +1,14 @@
+13. According to https://ephocks.atlassian.net/browse/DOC-1533[DOC-1533]:
```

Comment:
@lnewson TINY-8382

Reply:
This is not correct. The SaxParser linked is a Java implementation and not something we can use. The SaxParser that was used by the media plugin was the one in the public TinyMCE APIs: https://www.tiny.cloud/docs/api/tinymce.html/tinymce.html.saxparser/

It is now still using a TinyMCE public API, it's just our `DomParser` API instead since we've removed the custom HTML SaxParser implementation in 6.0 (https://www.tiny.cloud/docs/api/tinymce.html/tinymce.html.domparser/ is the equivalent from 5.x). So the change here is that the contents within media elements must be configured via the TinyMCE Schema (generally done by the `valid_elements`, `valid_children` or `extended_valid_elements` config). This will likely need mentioning in the migration guide as a minor note. I'm not sure if it should be in the release notes sorry.

Reply:
Also worth noting the parser we switched to is not our own, under the hood our `DomParser` API uses the browser parsing API of the same name
https://developer.mozilla.org/en-US/docs/Web/API/DOMParser

Reply:
Yeah I missed that here, but had mentioned that in another answer sorry (see Q15).

Reply:
Haven't read all the questions, so unsure if this is mentioned elsewhere, but: generally, I think it's worth mentioning the parser change in the Release Notes due to the reason we did it - which is security improvements. Regarding this particular aspect - the relationship between the parser and the media plugin - that I don't think we need to make particular note of.